lib/ex_aws/sts.ex

```diff
@@ -50,6 +50,27 @@ defmodule ExAws.STS do
     request(:assume_role_with_web_identity, params)
   end
 
+  @type assume_role_with_saml_opt ::
+          {:duration, pos_integer}
+          | {:policy, policy}
+
+  @doc "Assume Role with SAML"
+  @spec assume_role_with_saml(
+          principal_arn :: String.t(),
+          role_arn :: String.t(),
+          saml_assertion :: String.t(),
+          [assume_role_opt]
```

Comment:
I think this should be
```
[assume_role_with_saml_opt]
```

Reply:
Nice catch, thank you! If all looks good you can approve + merge this. We will probably have to get Ben to to publish the new version on hex as well. Also if you need to get in touch I have the same username on the Elixir Slack 👍